src/common/shared_config.h

```diff
@@ -11,8 +11,23 @@
 #define FW_UPDATE_ENABLE  0xAA
 #define FW_UPDATE_DISABLE 0x00
 
-// pin PA13 state
-#define APPENDIX_FLAG_MASK 0x01
+// appendix state
+typedef enum {
+    APNDX_EXISTS = 0, /**< Appendix not broken, signature authentication needed */
+    APNDX_BROKEN      /**< Appendix broken, no signature authentication needed */
+} APNDX_STATUS;
+
+// model_specific_flags bit locations from LSB
+typedef enum {
+    appednix_status = 0, /**< appendix status flag */
+    reserved_1,
+    reserved_2,
+    reserved_3,
+    reserved_4,
+    reserved_5,
+    reserved_6,
+    reserved_7
+} model_specific_flags;
```

Comment:
1. Where are the flags located in memory?
2. There is a typo in `appednix_status`
3. Why is the name of `APNDX_STATUS` capitalized and the name of `model_specific_flags` not?
4. The enum's cases `reserved_1`, `reserved_2`, etc are all global symbols. They should be prefixed with the enum's name.

Reply:
refer to usage "volatile data_exchange_t ram_data_exchange __attribute__((section(".boot_fw_data_exchange")));" in src/common/sys.cpp, line 16 for location in memory